config/src/main/resources/default-execution-parameters.json

```diff
@@ -5,5 +5,7 @@
   "proto": false,
   "url": null,
   "zipinput": null,
-  "exclude": null
+  "exclude": null,
+  "mintransfertime": 1,
```

Comment:
I think we should adopt a more excplicit format for our parameters name like filename__fieldname__qualifier
In that case `transfers__min_transfer_time__min` and `transfers__min_transfer_time__max`

Reply:
Keeping this in mind for future implementation of semantic validation of numeric fields in a distinct use case.